frontend/express/public/core/events/javascripts/countly.overview.views.js

```diff
@@ -1,32 +1,140 @@
 /* global countlyVue, countlyCommon, countlyEventsOverview,CV,app*/
-var EventsTable = countlyVue.views.BaseView.extend({
-    mixins: [countlyVue.mixins.i18n],
-    data: function() {
-        return {
-            scoreTableExportSettings: {
-                title: "Events",
-                timeDependent: true
+(function() {
+    var EventsTable = countlyVue.views.BaseView.extend({
+        mixins: [countlyVue.mixins.i18n],
+        data: function() {
+            return {
+                scoreTableExportSettings: {
+                    title: "Events",
+                    timeDependent: true
+                }
+            };
+        },
+        methods: {
+            onRowClick: function(params) {
+                app.navigate("#/analytics/events/key/" + params.name, true);
             }
-        };
-    },
-    computed: {
-        eventsTableRows: function() {
-            return this.$store.getters["countlyEventsOverview/detailEvents"];
         },
-    },
-    template: '#overview-tables-events'
-});
+        computed: {
+            eventsTableRows: function() {
+                return this.$store.getters["countlyEventsOverview/detailEvents"];
+            },
+        },
+        template: '#overview-tables-events'
+    });
 
-var EventsBreakdownHorizontalTile = countlyVue.views.BaseView.extend({
-    props: {
-        trend: {
-            type: String
+    var OverviewConfigureDrawer = countlyVue.views.create({
+        template: CV.T('/core/events/templates/configureEvents.html'),
+        data: function() {
+            return {
+                saveButtonLabel: CV.i18n('common.done'),
+                selectEvent: '',
+                selectProperty: '',
+            };
         },
-        change: {
-            type: String
+        computed: {
+            title: function() {
+                return CV.i18n("events.overview.events.configure.events");
+            },
+            eventProperties: function() {
+                return this.$store.getters["countlyEventsOverview/eventProperties"];
+            },
+            configureEventsList: function() {
+                return this.$store.getters["countlyEventsOverview/configureEventsList"];
+            },
+            selectedOverviewEvents: {
+                get: function() {
+                    return this.selectEvent;
+                },
+                set: function(selectedItem) {
+                    this.selectEvent = selectedItem;
+                    this.$store.dispatch('countlyEventsOverview/fetchEventProperties', selectedItem);
+                }
+            },
+            selectedOverviewEventProperties: {
+                get: function() {
```

Comment:
why use getter/setter here?
give your usage of the key you could have directly used the key as v-model provided it was defined in data and not in computed.

Reply:
yes that would be fine. Will do that :)